api/client-server/administrative_contact.yaml

```
@@ -134,9 +134,25 @@ paths:
         200:
           description: The addition was successful.
           examples:
-            application/json: {}
+            application/json: {
+                "submit_url": "https://example.org/path/to/submitToken"
```

Because the style is 2 spaces indentation, which applies to everything else too:

```suggestion
              "submit_url": "https://example.org/path/to/submitToken"
```